{{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/commands.py

```diff
@@ -2,23 +2,22 @@
 
 import subprocess
 
-from toil_container import ContainerCallJob
-from toil_container import ContainerShortArgumentParser
+from toil_container import ContainerArgumentParser
+from toil_container import ContainerJob
 
 from {{cookiecutter.project_slug}} import __version__
 
 
-class BaseJob(ContainerCallJob):
+class BaseJob(ContainerJob):
 
     """
-    A job base class that inherits from `ContainerCallJob`.
+    A job base class that inherits from `ContainerJob`.
 
-    This class provides, abstract methods `check_call` and `check_output` which
-    will use `docker`, `singularity` or `subprocess` to execute system calls.
+    Use `self.call` to run commands with docker, singularity or subprocess.
+    Learn more here https://github.com/leukgen/toil_container.
 
     Arguments:
-        options (argparse.Namespace): an `argparse.Namespace` with pipeline
-            options obtained with `ContainerShortArgumentParser`.
+        options (argparse.Namespace): `ContainerArgumentParser` namespace.
```

Comment:
Just leave the `Attribute`, no? the object has `self.options`. I don't understand what you mean with the `Argument`? argument of the constructor `__init__`? No need to document that